content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -1010,6 +1028,31 @@ is registered to handle that kind of data source. When a suitable populator is i
 responsibility of that populator controller to report Events that relate to volume creation and issues during
 the process.
 
+### Using Cross Namespace Volume Data Source
+
```

Comment:
Can you add the feature gate tag here, and also add an example of the ReferenceGrant?

Reply:
Add feature gate tag and example.